pkgs/development/libraries/x265/default.nix

```diff
@@ -34,9 +34,13 @@ let
     sha256 = "1jzgv2hxhcwmsdf6sbgyzm88a46dp09ll1fqj92g9vckvh9a7dsn";
   };
 
+  patches = stdenv.lib.optionals stdenv.hostPlatform.isAarch64 [
+    ./0001-dynamicHDR10-update-CMakeLists.txt-for-aarch64-suppo.patch
```

Comment:
Please don't bundle patches in nixpkgs, if they can be sent upstream, and `fetchpatch`ed in here.

When doing so, please also add a comment pointing to the upstream PR.